.github/workflows/turbopack-test.yml

```diff
@@ -425,87 +173,6 @@ jobs:
         working-directory: crates/turbopack-ecmascript-runtime/js
         run: pnpm run check
 
-  rust_lint:
-    needs: [determine_jobs]
-    if: needs.determine_jobs.outputs.rust == 'true'
-    name: Rust lints
-    runs-on:
-      - "self-hosted"
-      - "linux"
-      - "x64"
-      - "metal"
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-
-      - name: Setup Rust
-        uses: ./.github/actions/setup-rust
-        with:
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
-
-      - name: Run cargo fmt check
-        run: |
-          cargo fmt --check
-
-      - name: Check Cargo.toml formatting (taplo)
-        run: npx @taplo/cli@0.5.2 format --check
-
-      - name: Check licenses
-        uses: EmbarkStudios/cargo-deny-action@v1
-        with:
-          command: check licenses
-
-  turborepo_rust_check:
-    needs: [determine_jobs]
-    # We test dependency changes only on main
-    if: |
-      (needs.determine_jobs.outputs.rust == 'true' && needs.determine_jobs.outputs.turborepo_rust == 'true') ||
-      needs.determine_jobs.outputs.cargo_on_main == 'true' ||
-      needs.determine_jobs.outputs.cargo_only == 'true'
-    name: Turborepo rust check
-    runs-on:
-      - "self-hosted"
-      - "linux"
-      - "x64"
-      - "metal"
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-
-      - name: Setup Turborepo Environment
-        uses: ./.github/actions/setup-turborepo-environment
-        with:
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
-
-      - name: Run cargo check
-        run: |
-          cargo groups check turborepo-libraries --features rustls-tls
-
-  turborepo_rust_clippy:
-    needs: [turborepo_rust_check]
-    name: Turborepo rust clippy
-    runs-on:
-      - "self-hosted"
-      - "linux"
-      - "x64"
-      - "metal"
-    steps:
-      - name: Checkout
-        uses: actions/checkout@v3
-
-      - name: Setup Turborepo Environment
-        uses: ./.github/actions/setup-turborepo-environment
-        with:
-          github-token: "${{ secrets.GITHUB_TOKEN }}"
-
-      - name: Run cargo clippy
-        run: |
-          cargo groups clippy turborepo-libraries --features rustls-tls -- --deny clippy::all
-
-      - name: Run ast-grep lints
```

Comment:
Is this accounted for in Turbopack still?

Reply:
son confused about what this diff is showing... this is clearly in the branch: https://github.com/vercel/turbo/blob/440dc9d4caf0183a1d5391da42b0be1f50896ce6/.github/workflows/turbopack-test.yml#L229